### PR TITLE
I update the contributing docs more detailed and easy understand

### LINKED ISSUE #82: Starknet Scaffold Documentation
Amazing job we've done thus far, but we need a great documentation, people can reference as there's not so much we can cover in the Readme.

**This issue will require 8 persons, with great writing skills. The documentation will include the following sections:**

- [x] Introduction - @Immanuelolivia1 
- [x] Quickstart/installation - @Joy-Adah 
- [x] UI components - @Oshioke-Salaki 
- [x] A tour guide of the different features [Deployer, Burner, Faucet, Wikipedia, Converter] - @MarvyNwaokobia 
- [x] A breakdown of the scripts - @kateberryd 
- [x] Spinning up and using devnet - @od-hunter 
- [x] Starknet Address Book - @Gift-Naomi 
- [x] Contributing to Starknet Scaffold - @mimiprosper

### PR DESCRIPTION
**Pull request by mimiprosper**
## Description

Contributing to Starknet Scaffold.

I update the contributing docs for Starknet Scaffold for more detailed and easy understand

### CODE REVIEW

File: docs/src/chapter_8.md [hunk 1]
```diff
@@ -1,6 +1,165 @@
-# Contributing to Starknet Scaffold
+# Scaffold-ETH 2 Contributing Guide
 
-Guide a first time contributor on how to contribute to the scaffold project.
-Resources:
-- https://github.com/argentlabs/Starknet-Scaffold/blob/main/CONTRIBUTING.md
-- https://docs.scaffoldeth.io/contributing/
+Welcome to the Scaffold-ETH 2 contributing guide! Thank you for investing your time in contributing to our project. This guide aims to provide a comprehensive overview of the contribution workflow, ensuring an effective and efficient process for everyone involved.
```
Comment (Darlington02) [inline docs/src/chapter_8.md:3]: Starknet-Scaffold not Scaffold-ETH2

File: docs/src/chapter_8.md [hunk 2]
```diff
@@ -1,6 +1,165 @@
...
+
+### About the Project
+Scaffold-ETH 2 is a minimal and forkable repository providing builders with a starter kit to develop decentralized applications (dApps) on Ethereum. For an overview of the project, please read the [README](README.md) to get an overview of the project.
```
Comment (Darlington02) [inline docs/src/chapter_8.md:6]: Starknet Scaffold!

**Review by Darlington02 — CHANGES_REQUESTED** [review]
Please rewrite the whole thing. You are writing a guide for the Starknet Scaffold project built on Starknet not scaffold-eth-2.

Just simply copy what we already have here, with some minimal updates as you see fit:
https://github.com/argentlabs/Starknet-Scaffold/blob/main/CONTRIBUTING.md

File: docs/src/chapter_8.md (hunk 2 above)
Comment (Darlington02) [reply docs/src/chapter_8.md:6]: and don't close the issue from your PR, leave the Related issue empty

**Comment by mimiprosper** [comment]
Goodmorning Boss,

I have taken note of the instruction. I would  fix them right away.

Thanks.

**Comment by mimiprosper** [comment]
Contributing to Starknet Scaffold.

I have made the correction and updates.

Related Issue #82

**Comment by mimiprosper** [comment]
I have update my comments so that i don't mention the issue

**Comment by Darlington02** [comment]
> Contributing to Starknet Scaffold.
> 
> I have made the correction and updates.

Ty ser! merged

**Pull request merged**